proto/kiichain/feeabstraction/v1beta1/params.proto

```
@@ -0,0 +1,12 @@
+syntax = "proto3";
+package kiichain.feeabstraction.v1beta1;
+
+import "gogoproto/gogo.proto";
```

Buf (1.55.1)

import "gogoproto/gogo.proto": file does not exist

(COMPILE)